Documentation/api-reference.md

```diff
@@ -128,6 +132,69 @@ Edge.sendEvent(experienceEvent: experienceEvent) { (handles: [EdgeEventHandle])
 }
 ```
 
+##### Example with DatastreamIdOverride
+```swift
+// Create Experience event from dictionary
+var xdmData : [String: Any] = ["eventType" : "SampleXDMEvent",
+                              "sample": "data"]
+let experienceEvent = ExperienceEvent(xdm: xdmData, datastreamIdOverride: "SampleDatastreamId")
+```
+```swift
+// Example 1 - send the Experience event without handling the Edge Network response
+Edge.sendEvent(experienceEvent: experienceEvent)
+```
+```swift
+// Example 2 - send the Experience event and handle the Edge Network response onComplete
+Edge.sendEvent(experienceEvent: experienceEvent) { (handles: [EdgeEventHandle]) in
+  // Handle the Edge Network response
+}
+```
+
+##### Example with DatastreamConfigOverride
+```swift
+// Create Experience event from dictionary
+var xdmData : [String: Any] = ["eventType" : "SampleXDMEvent",
+                              "sample": "data"]
+
+ let configOverrides: [String: Any] = [
+                                        "com_adobe_experience_platform": [
+                                          "datasets": [
+                                            "event": [
+                                              "datasetId": "eventDatasetIdOverride"
```

Comment:
for consistency can we use:
SampleEventDatasetIdOverride
SampleProfileDatasetIdOverride
SamplePropertyToken